includes/class.llms.post-types.php

```diff
@@ -102,6 +105,27 @@ public static function add_thumbnail_support() {
 
 	}
 
+	/**
+	 * De-register protected post types from wp-sitemap.xml
+	 *
+	 * @since [version]
+	 *
+	 * @param WP_Post_Type[] $post_types Array of post types.
+	 * @return WP_Post_Type[]
+	 */
+	public static function deregister_sitemap_post_types( $post_types ) {
+
```

Comment:
can we add a filter here to which we pass the list of post types to be ignored, so that from our add-ons we use our own filter?
I like it more. :)

Reply:
I thought about that but like... does that save us any work, really? I've checked our other add-ons and we don't actually have an issue with any of our other add-ons, except groups, which actually requires a more complicated and conditional solution based on the way groups work (see: https://github.com/gocodebox/lifterlms-groups/issues/126)

Reply:
I see, yeah pa posts are not public(ly queryable) I see.
Well I was even thinking that maybe someone would have lessons available in sitemaps? Why? yeah good question, free lessons?
But yeah, nevermind we'll add it later if requested.